documentation/AuthServer.md

```diff
@@ -25,56 +53,109 @@ The document types used by the authorization server include:
+| `type` | String | The document type for storing privileged prefs creators. | The value must be set to "privilegedPrefsCreatorAuthorization". |
 | `clientId` | String | The client id that this token is assigned to. | None |
-| `accessToken` | String | The access token used to retrieved the protected user preferences. | None |
-| `allowAddPrefs` | Boolean | Whether the client is allowed to add new preferences. | true |
+| `accessToken` | String | The access token that indicates the client has privilege to add preferences. | None |
 | `revoked` | Boolean | Whether this token has been revoked. | false |
 
+#### Web Prefs Consumer Clients
+
+| Option | Type | Description | Default |
+| ------ | ---- | ----------- | ------- |
+| `id` | String | The client id. Can be a UUID or any unique string. This value is saved into `_id` field in CouchDB/PouchDB. | None |
+| `type` | String | The document type for storing client information. | The value must be set to "webPrefsConsumerClient". |
+| `name` | String | The client name. | None |
+| `oauth2ClientId` | String | The unique identifier issued to a registered OAuth2 client by the authorization server. | None |
+| `oauth2ClientSecret` | String | Confidential shared secret between the client and the authorization server, used to verify the identity of the client. | None |
+| `redirectUri` | String | The URL on client's site where users will be sent after authorization.| None |
+
+#### Web Prefs Consumer Authorizations
+
+| Option | Type | Description | Default |
+| ------ | ---- | ----------- | ------- |
+| `id` | String | The auth code authorization id. Can be a UUID or any unique string. This value is saved into `_id` field in CouchDB/PouchDB. | None |
+| `type` | String | The document type for storing GPII tokens. | The value must be set to "webPrefsConsumerAuthorization". |
+| `gpiiToken` | String | The GPII token that the authorization is associated with. | None |
+| `clientId` | String | The client id that the authorization is associated with. | None |
+| `redirectUri` | String | The URL on client's site where users will be sent after authorization. | None |
+| `accessToken` | String | The access token used to retrieve the protected user preferences. | None |
+| `selectedPreferences` | Object | The preferences that the user has granted permissions to the client to retrieve. | None |
+| `revoked` | Boolean | Whether this authorization has been revoked. | false |
+
+#### Authorization Codes
+
+| Option | Type | Description | Default |
+| ------ | ---- | ----------- | ------- |
+| `id` | String | The authorization code id. Can be a UUID or any unique string. This value is saved into `_id` field in CouchDB/PouchDB. | None |
+| `type` | String | The document type for storing GPII tokens. | The value must be set to "authCode". |
+| `authCodeAuthorizationId` | String | The auth code authorization that this code is associated with. | None |

```

Comment:
I think `authCodeAuthorizationId` should be `authorizationId`:

https://github.com/cindyli/universal/blob/GPII-2435/gpii/node_modules/gpii-oauth2/gpii-oauth2-datastore/src/DbDataStoreUtils.js#L450